docs/source/cli_help/traverse.rst

```diff
@@ -65,6 +65,13 @@ It will pick up the walk from the current branch (unless ``--start-from=`` or ``
 Unlike with e.g. ``git rebase``, there is no special ``--continue`` flag, as ``traverse`` is stateless
 (doesn't keep a state of its own like ``git rebase`` does in ``.git/rebase-apply/``).
 
+``Traverse`` rebase/merge/push behaviour can also be customized for each branch separately using branch annotation qualifiers.
+There are ``rebase=no`` and ``push=no`` qualifiers that can be used to opt out of default behaviour (rebasing and pushing),
+there is ``merge=no`` qualifier that can be used to not use merge for certain branch when executing traverse with [-M|--merge] option
+and also a ``merge=yes`` qualifier that can be used to force ``traverse`` to use merge instead of rebase for certain branch (you don't need to provide [-M|--merge] flag).
```

Comment:
Okay.... so given that I do everything I can to **discourage** users from merging branches due to https://github.com/VirtusLab/git-machete#can-i-use-git-merge-for-dealing-with-stacked-prs ... I'd drop support for `merge=yes` entirely. There hasn't been any request for such feature (unlike with `...=no`), and it'd have adverse effect on UX (since the folks will end up doing more merges that they'd do otherwise).

I'm even hesitant about `merge=no` TBH. It might have the side effect of presenting merge (`git machete traverse --merge` as an option of an equal standing with rebase... which it definitely isn't. Also, `merge=no` hasn't been requested by anyone either IIRC 🤔

Reply:
@PawelLipski Well, i see your point, in that case we can definitely drop `merge=yes`, when it comes to `merge=no`, we added it to be extra precise but it's not 100% required so i guess it's up to you.

Reply:
Yes, let's remove both `merge=no` and `merge=yes`. And maybe reconsider re-adding `merge=no` once requested (but even then, I'll still probably be hesitant)